node/package.json

```diff
@@ -15,6 +15,7 @@
     "@types/uuid": "^8.3.4",
     "@typescript-eslint/eslint-plugin": "^5.49.0",
     "@typescript-eslint/parser": "^5.49.0",
+    "cassandra-driver": "^4.6.4",
```

Comment:
This is for testing only.
It's in devDependencies and won't be included in official release.